distribution/docs/src/main/resources/content/_configuring/map-layers-intrigue.adoc

```diff
@@ -36,6 +36,6 @@ To remove all map layers, select *RESET*.
 
 .Map Layer Advanced Configuration
 Select *Advanced Configuration* to edit the JSON-formatted configuration directly.
-See <<org.codice.ddf.catalog.ui.config,${catalog-ui} Configurations>> for examples of map layer configurations.
+See <<org.codice.ddf.catalog.ui.config,Catalog UI Search Configurations>> for examples of map layer configurations.
```

Comment:
`${catalog-ui}` resolves to Intrigue, but the configuration is named `Catalog UI Search`.
![image](https://user-images.githubusercontent.com/8041246/39154794-5cc205ba-4704-11e8-94cc-97ddce47240d.png)